definitions/variable/emissions/emissions.yaml

```
@@ -248,21 +341,25 @@
     unit: "{Level-3 Species}"
 
 - Emissions|{Level-2 Species}|Waste:
-    description: Emissions of {Level-2 Species} from waste (IPCC category 6)
-    unit: "{Level-2 Species}"
-
-- Emissions|{Level-2 Species}|Fossil Fuel Fires:
-    description: Emissions of {Level-2 Species} from fossil fuel fires (IPCC category 7)
+    description: Emissions of {Level-2 Species} from waste of fossil-based products
+      through incineration or decomposition (IPCC 1996 category 6; IPCC 2006 category 4), 
+      not including emissions from organic waste handling and decay
     unit: "{Level-2 Species}"
 
 - Emissions|{Level-2 Species}|Other:
-    description: Emissions of {Level-2 Species} from other sources
+    description: Emissions of {Level-2 Species} from other sources (IPCC 1996 category 7;
+      IPCC 2006 category 5B) including fossil fuel fires and any sources or sinks that
+      cannot be accommodated in other categories
+      notes: This will be treated as a flux from the fossil reservoir to the atmosphere in climate models.
```

This will fix the failing test.
```suggestion
    notes: This will be treated as a flux from the fossil reservoir to the atmosphere in climate models.
```